schema/resource.go

```diff
@@ -87,56 +78,48 @@ func (r *Resource) GetItem() any {
 	return r.Item
 }
 
-func (r *Resource) GetValues() CQTypes {
-	return r.data
-}
-
-func (r *Resource) Columns() []string {
-	return r.Table.Columns.Names()
-}
-
 //nolint:revive
-func (r *Resource) CalculateCQID(deterministicCQID bool) error {
-	if !deterministicCQID {
-		return r.storeCQID(uuid.New())
-	}
-	names := r.Table.PrimaryKeys()
-	if len(names) == 0 || (len(names) == 1 && names[0] == CqIDColumn.Name) {
-		return r.storeCQID(uuid.New())
-	}
-	slices.Sort(names)
-	h := sha256.New()
-	for _, name := range names {
-		// We need to include the column name in the hash because the same value can be present in multiple columns and therefore lead to the same hash
-		h.Write([]byte(name))
-		h.Write([]byte(r.Get(name).String()))
-	}
-	return r.storeCQID(uuid.NewSHA1(uuid.UUID{}, h.Sum(nil)))
-}
-
-func (r *Resource) storeCQID(value uuid.UUID) error {
-	b, err := value.MarshalBinary()
-	if err != nil {
-		return err
-	}
-	return r.Set(CqIDColumn.Name, b)
-}
+// func (r *Resource) CalculateCQID(deterministicCQID bool) error {
```

Comment:
Probably need to bring this back or delete it if it was unused

Reply:
I brought it back with `panic("not implemented")` as `Resource` is here just for some of the table definition to compile but it is not used anywhere as v3 still support only destination.